### DISCUSSION TITLE
Using go2rtc instead of mediamtx

### DISCUSSION THREAD

**Discussion by lexusburn** [fuatakgun/eufy_security#878] [Ideas]
Currently i'm using go2rtc as a solution for low latency video streaming.

If i want to use this very nice plugin, i have to use mediamtx instead, because both are using the default ports for rtsp/rtmp and should do the same. 

Is there any possibility to change from mediamtx to go2rtc?

**Comment by fuatakgun** [comment]
Not sure how go2rtc will make it faster, any suggestions to convert bytes into a stream using go2rtc?

**Reply by pmuir** [reply]
See https://github.com/AlexxIT/go2rtc#incoming-sources - so exactly the same way you do it with simple-rtsp-server just a different output url. If you can tell me how you would like the config to look I can try to do a PR.
